packages/hasura-storage-js/src/hasura-storage-api.ts

```diff
@@ -116,12 +109,9 @@ export class HasuraStorageApi {
     return uploadheaders
   }
 
-  private generateAuthHeaders():
-    | { Authorization: string }
-    | { 'x-hasura-admin-secret': string }
-    | null {
+  private generateAuthHeaders(): HeadersInit | undefined {
     if (!this.adminSecret && !this.accessToken) {
-      return null
+      return undefined
```

Comment:
Why did you change it to `undefined`?

Reply:
because `generateAuthHeaders()` is now used with the `headers` option of `fetch`, that accepts either an `Record<string,string>` or `undefined`